recipes/pytest-raises/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pytest-raises" %}
+{% set version = "0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4710a8921b0b7425cd2073d696862ecf99922ef522e681cc63a52008112d63d9
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```

Comment:
Looks like this package has an entry point defined in setup.py. If so, please add that here in the build section too, e.g.:
entry_points:
  - raises = pytest_raises.pytest_raises
and test for "raises" in the `test: commands` section.

You can read more about entry points here, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points.

Reply:
This is confusing (to me anyway) this is a pytest plugin of some sort. I think the entry_point is for pytest to use, rather than a command.

You usually use this via a pytest fixture: `@pytest.mark.raises()` or `@pytest.mark.setup_raises`.

So I don't think there is a command installed. In any case, if you pip install this one, you don't get a "raises" command.

I also just looked at the package again -- it has tests, but they are not in a package, so do not run with:
pytest --pyargs pytest_raises

So the import test may be all we can do.

Reply:
Looked a bit more. It is not a "console_scripts" entry point:

```
        entry_points={
            'pytest11': [
                'raises = pytest_raises.pytest_raises',
            ],
```
it's a pytest specific one. 

So I think we're good here.

Reply:
I agree I think it is good to go for now. Any problem we can change on the feedstock